.git-blame-ignore-revs

```diff
@@ -0,0 +1,3 @@
+# .git-blame-ignore-revs
```

Comment:
we could probably use the same file under `shared-web` as well

Reply:
https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view
The file must be on root dir

Reply:
that'll translate to the root directory for https://github.com/determined-ai/shared-web